src/styles/about.scss

```diff
@@ -4,5 +4,5 @@
   @extend .side-nav;
   overflow: hidden;
   position: inherit;
-  width: auto;
+  width: 250px;
```

Comment:
This change is breaking mobile view:

<img width="382" alt="Screenshot 2021-06-18 at 11 34 11" src="https://user-images.githubusercontent.com/5843270/122532391-1f29c600-d029-11eb-92f7-d0d360209ffe.png">